### DISCUSSION TITLE
Bed Levelling poor accuracy in MOD

### DISCUSSION THREAD

**Discussion by Grogamanda1** [xblax/flashforge_ad5m_klipper_mod#151] [Q&A]
Hi all ,

Im having some issues with the ABL in the mod - Cant seem to get a good first level after calibration. If I reboot to stock firmware and run Bedlevelling I get perfect 1st layer. 

Does the mod and stock firmware use different ABL methods?

Cheers

**Comment by consp** [comment]
> Does the mod and stock firmware use different ABL methods?

Flashforge does some trickery with the z-offset. It defaults to -0.25, but I had to change it to -0.205 to get good results. Maybe that also works for you (ymmv).

**Comment by AJolly** [comment]
What does it look like if you run PROBE_ACCURACY ?

**Comment by Arnaud3767** [comment]
Hello,
I performed PROBE_ACCURACY and here are my results:

probe accuracy results: maximum 53.152500, minimum -2.805000, range 55.957500, average 2.802750, median -2.788750, standard deviation 16.783253

it's bad?

**Comment by Arnaud3767** [comment]
sorry, I was scared, I performed PROBE_ACCURACY at the corner of the pei.

Your results in the middle:
probe accuracy results: maximum -2.860000, minimum -2.892500, range 0.032500, average -2.874250, median -2.871250, standard deviation 0.009357

**Reply by consp** [reply]
>  standard deviation 0.009357

That's fine, it's pretty much mechanical slack somewhere.

16 on the other hand would be massive and probably a measurement error.

My resutls (after I removed all residue from the head):

```
// probe accuracy results: maximum -1.412500, minimum -1.432500, range 0.020000, average -1.422250, median -1.422500, standard deviation 0.006562
```

Note that the offset is a bit different since I changed it.

**Reply by Arnaud3767** [reply]
Thanks for your feedback. 
And a big thank you to everyone involved in creating klipper for this 3d printer
